package.json

```diff
@@ -25,7 +25,9 @@
   },
   "devDependencies": {
     "all-contributors-cli": "^6.16.1",
-    "lerna": "^3.22.1"
+    "lerna": "^3.22.1",
+    "set-tz": "0.2.0",
```

Comment:
This allows me to fix the timezone, so that the tests work when run locally, regardless of which OS and which timezone the developers machine is using.

Reply:
Dropped after rework.